api/URLPattern.json

```
@@ -96,6 +96,43 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "ignoreCase": {
```

To keep with traditional naming conventions, we should tweak this feature name slightly:

```suggestion
        "ignoreCase_option": {
```